README.md

```diff
@@ -36,6 +36,70 @@ over 100,000 metric values per second with less than 0.2% overhead.
 
 # Building the OVIS / LDMS source code
 
+## Pre-built containers
+
+You may skip the building process and use the pre-built containers from the
+following list. Please follow the corresponding links for more information on
+the images.
```

Comment:
A suggestion for an alternative introduction.

You may avoid the hassle of building LDMS from scratch by leveraging containerized deployments. Here's a collection of LDMS container images available for you to pull an run. Each image offers a specific set of functionalities to suit your needs. Refer to the corresponding links for detailed information on each image.